app/models/firm.rb

```diff
@@ -1,3 +1,94 @@
 class Firm < ActiveRecord::Base
+  has_and_belongs_to_many :in_person_advice_methods
+  has_and_belongs_to_many :other_advice_methods
+  has_and_belongs_to_many :initial_advice_fee_structures
+  has_and_belongs_to_many :ongoing_advice_fee_structures
+  has_and_belongs_to_many :allowed_payment_methods
+  has_and_belongs_to_many :investment_sizes
+
+  belongs_to :initial_meeting_duration
+
   has_many :advisers
+
+  attr_accessor :percent_total
+
+  validates :email_address,
+            presence: true,
```

Comment:
Can we align the arguments for all of these validators as per the rest of the project. i.e.

```ruby
validates :email_address,
  presence: true,
  something: false,
  bleh: :meh
```